SDKバージョン等もろもろ更新

yabuchin opened the pull request:
SDKバージョンを最新にして、Android Studio 3でビルド出来るようにしました。
また、アイコンも丸いアイコンに変更しました。

Commit:
SDKバージョン等もろもろ最新のものに更新

Commit:
丸いアイコンに修正

yabuchin committed:
Merge pull request #62 from benkyokai/feature/yabu/round-icon

amake commented:
jumblr と GifImageView サブモジュールの変更はプッシュされていない（tumpaca リポジトリはリモートに存在しないコミットを参照している）ようなので一度プッシュをお願いします。

yabuchin commented:
すいません、GifImageViewをプッシュしました。jumblrは変更がありません。

amake commented:
Pixel 2 のランチャーだと、アプリアイコンの形状を以下の中から選択できます。

- システムデフォルト（Pixel 2 は circle）
- Square
- Rounded square
- Squircle
- Teardrop

いろんな形に対応できるのはたぶん dynamic icon ってやつですが、そこまで対応しなくても別にいいと思います。たぶん、[ここ](https://developer.android.com/about/versions/nougat/android-7.1.html#circular-icons)にあるように既存の `android:icon` に加え `android:roundIcon` に今回の丸いアイコンを追加すればその2パターンはカバーできると思います。

amake commented on app/build.gradle:
```diff
@@ -34,9 +34,8 @@ ext {
-    compileSdkVersion 25
-    buildToolsVersion '25.0.2'
+    version "1.2.2"
+    compileSdkVersion 26
```
`targetSdkVersion` を 26 に合わせてもいいと思いましたが、あえてそうしないことにしたんですか？

amake commented:
↑アイコンの話ですが、もし丸いアイコンだけ提供したいという意図的な仕様であれば無視してください。

amake commented:
このブランチでビルドしてテストしてみたら、プロフィールアイコンが表示されないようです。今の master では表示されます。このPRの中で直さないという判断もありですが、この問題は把握していますか？

Commit:
jumblr へ参照するコミットが古くなっていたので https://github.com/benkyokai/tumpaca/pull/60 の問題の一部が再現していた。

- #60 の問題はクラッシュする問題であったが、クラッシュ自体は tumpaca 側のコードで防がれるようになっていて、単にアイコンが表示されなくなっていた。

Commit:
Adaptive Icon に対応

amake reviewed (approved):
lgtm
